k8s/nginx/container/Dockerfile

```diff
@@ -7,5 +7,5 @@ RUN apt-get update \
     && apt-get clean
 COPY nginx.conf.template /etc/nginx/nginx.conf
 COPY nginx_entrypoint.bash /
-EXPOSE 80 443 27017
+EXPOSE 80 81 443 444 27017
```

Comment:
Out of interest, how come it could not be a separate `location` block within the same `server` as the HTTP API?